src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -556,6 +617,14 @@ def update_container_settings(cmd, resource_group_name, name, docker_registry_se
         update_app_settings(cmd, resource_group_name, name, settings, slot)
     settings = get_app_settings(cmd, resource_group_name, name, slot)
 
+    if multicontainer_config_file and multicontainer_config_type:
+        encoded_config_file = _get_linux_multicontainer_encoded_config_from_file(cmd, resource_group_name,
+                                                                                 name, multicontainer_config_file)
+        linux_fx_version = _format_linux_fx_version(encoded_config_file, multicontainer_config_type)
+        update_site_configs(cmd, resource_group_name, name, linux_fx_version=linux_fx_version)
+    else:
```

Comment:
should not this be `multicontainer_config_file or multicontainer_config_type`? Otherwise, setting an unrelated config like `docker_custom_image_name` would trigger this warn.